documentation/modules/appendix_crds.adoc

```diff
@@ -781,10 +783,67 @@ It must have the value `password` for the type `KafkaJmxAuthenticationPassword`.
 |string
 |====
 
+[id='type-JmxTransSpec-{context}']
+### `JmxTransSpec` schema reference
+
+Used in: xref:type-KafkaJmxOptions-{context}[`KafkaJmxOptions`]
+
+
+[options="header"]
+|====
+|Property                  |Description
+|outputDefinitions  1.2+<.<|Defines the output hosts that will be referenced later on. For more information, see `proc-jmxtrans-deployment.adoc`.
```

Comment:
The link won't work. We tend to reference the schema from the main part of the guide rather than the other way around. So maybe we can drop this link and refer back to here from the procedure? Maybe also we should say more about 'output hosts' in the procedure.

Reply:
I mentioned above, but something like:

For more information on these properties see, xref:type-JmxTransSpec-reference[`JmxTransSpec` schema reference].